packages/angular_devkit/build_angular/src/builders/karma/tests/options/exclude_spec.ts

```diff
@@ -54,7 +54,7 @@ describeKarmaBuilder(execute, KARMA_BUILDER_INFO, (harness, setupTarget) => {
       expect(result?.success).toBeTrue();
     });
 
-    it(`should exclude spec that matches the 'exclude' pattern prefixed with a slash`, async () => {
+    fit(`should exclude spec that matches the 'exclude' pattern prefixed with a slash`, async () => {
```

Comment:
```suggestion
    it(`should exclude spec that matches the 'exclude' pattern prefixed with a slash`, async () => {
```

Reply:
Argh, forgot that I did it on two levels. Thanks!